scripts/remove/netdata.sh

```diff
@@ -1,9 +1,9 @@
 #! /bin/bash
 # Netdata uninstaller for swizzin
 
-systemctl disable -q netdata > /dev/null 2>&1
-systemctl stop -q netdata
-rm -rf $(which netdata)
-rm -rf /etc/netdata
-rm -rf /var/log/netdata
+"${NETDATA_PREFIX}"/usr/libexec/netdata/netdata-uninstaller.sh --yes --env /etc/netdata/.environment -f >> $log 2>&1 || {
```

Comment:
Why is `${NETDATA_PREFIX}` here if there is no option to define it?

Reply:
It could be actually recorded in the ENV

Reply:
Actually it cannot, I'm removing this.